Added Jest icon #1326

nexus0212 opened the pull request:
**Issue: #1326**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Color was picked from the website element class `.headerTitleWithLogo` [1]

---
[1] https://jestjs.io/

ericcornelissen commented on icons/jest.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Jest icon</title><path d="M12 9.7l-1.4 2.9c.5.4.8 1 .9 1.7h1.1c.1-.6.4-1.2.9-1.6-.1 0-1.5-3-1.5-3z" fill="none"/><path d="M12.9 13.4L15.6 8l2.6 5.4c.1 0 .2-.1.3-.1L21 5.9H10.1l2.5 7.4c.1 0 .2 0 .3.1zm-.9 4.7c1 0 1.8-.8 1.8-1.8 0-.4-.1-.7-.3-1-.1-.1-.1-.2-.2-.3-.1-.1-.2-.1-.2-.2H13h-.1-.1-.1-.1-.1-.1-.1-.4-.1-.1-.1c-.8.2 ...
```
Looking at the quality of the icon it seems you've optimized a bit too much (look at the open circles at the bottom)

ericcornelissen replied:
It seems there is still a `fill="none ` left, could you remove that?

ericcornelissen replied:
Can you try to compound the paths in this SVG? If you need help with this let me know 😄

nexus0212 commented:
I fixed all bugs 💃. @ericcornelissen, I will be grateful to check that everything is okey.

ericcornelissen commented:
Great work @nexus0212. I have two more questions/suggestions:

1. Can you update the link to either the website (https://jestjs.io/) or repository (https://github.com/facebook/jest) both contain SVG versions of the log as well, and they are more likely to exist in the future than the link you selected.
1. For the color you have selected `#15c213` based on the SVG in the link you provided. However, I feel the brand color of Jest is more red than green, would you agree? For example the red in [this SVG](https://github.com/facebook/jest/blob/master/website/static/img/jest.svg): `#c21325` 🤔

nexus0212 commented:
> 2\. For the color you have selected `#15c213` based on the SVG in the link you provided. However, I feel the brand color of Jest is more red than green, would you agree? For example the red in [this SVG](https://github.com/facebook/jest/blob/master/website/static/img/jest.svg): `#c21325` 🤔

Of course I replace the color. In the first time I sugested the color from typographic logo Jest on the website https://jestjs.io/ 🤨

---
- [x] Updated the link
- [x] Changed color `#15c213` -> `#c21325`

ericcornelissen commented:
Also, can I give you a tip about committing @nexus0212? Because you updated the commit and force pushed all changes on this PR the related issues now looks like this :arrow_down:. Not necessarily a problem but it gets a bit messy.

![image](https://user-images.githubusercontent.com/3742559/56276973-2df45580-610c-11e9-9178-02c457873621.png)

At least for this project it is easier to just create a new commit. This makes your life easier as you don't have to write comments like this :arrow_down: Instead that information will just be in the commit history. You don't have to care about cluttering the commit history as we use [squash commits](https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits) for Pull Requests like this 😉 

![image](https://user-images.githubusercontent.com/3742559/56277139-6f850080-610c-11e9-91da-af39c9c2b071.png)

I hope that helps (don't hesitate to ask for clarification). Again, it is just some advice and you're doing great work! It is much appreciated 😃
